<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>928</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_change">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>340</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="range_start">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_slow">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="state">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>270</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="speed">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x 1.0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_baby">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_done">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>530</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_fast">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_search">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_continue">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>연속 재생</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_video">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="range_end">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_view">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>631</width>
      <height>451</height>
     </rect>
    </property>
    <property name="text">
     <string>이 곳에서 영상이 출력됩니다.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_now">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>580</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>현재 프레임 위치 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sensor">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>630</y>
      <width>861</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>센서(레이블링) 그래프</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_video">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>750</y>
      <width>861</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>영상 움직임 그래프</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_stop">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>영상 멈춤</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>860</y>
      <width>701</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
